util/constants.py

```diff
@@ -7,6 +7,14 @@
 
 MAX_ALLOCATE = 10000.0
 
+DO_PUBREWARDS = False
+
+# used by _rankBasedAllocate() as part of reward function
+DO_RANK = True
+RANK_SCALE_OP = "LOG"  # can be: LIN, POW2, POW4, LOG, SQRT
```

Comment:
It's good to see that we have the option to change the type of ranking scale that's used. Or ranking could in fact be turned of if we decided to move in a different direction.

Reply:
I think we need a clear success criteria so that we can objectively decide which ranking scale is best and how many assets should receive rewards.